src/domain/repositories/UserInfoNotificationsRepository.ts

```diff
@@ -0,0 +1,6 @@
+import { Async } from "domain/entities/Async";
+import { UserInfoNotification } from "domain/entities/UserInfoNotification";
+
+export interface UserInfoNotificationsRepository {
+    get(userDetailsPath: string, emailContentJsonPath: string): Async<UserInfoNotification>;
```

Comment:
The idea for the domain repository is to be as general as possible. We don't know where the data will come from, so it's better not to say "json" or "path". Proposal: 

`get(userDetailsId: string, emailContentsId: string): Async<UserInfoNotification>;`

And then in the data repository, the arguments can have the full name, and be interpreted as paths.